src/Twig/EzTabRenderExtension.php

```diff
@@ -22,19 +21,17 @@ public function getFunctions()
         ];
     }
 
-    public function crAddEzTabs(Environment $twig, string $param1, array $options = [], string $template = '') {
+    public function crAddEzTabs(Environment $twig, string $param1, array $options = [], string $template = '')
+    {
         $function = $twig->getFunction('ez_platform_tabs');
-        if ($function !== false) {
+        if (false !== $function && null !== $function) {
```

Comment:
Wouldn't `instanceof` be a more appropriate test in that case ?

Reply:
Changed